laser_slam_tools/src/octomap_to_point_cloud.cc

```diff
@@ -0,0 +1,48 @@
+#include <glog/logging.h>
```

Comment:
I see! We have always used cpp so far then we can stick to that for now (otherwise we should change everything).